app/src/main/java/net/osmtracker/osm/UploadToOpenStreetMapTask.java

```diff
@@ -131,7 +131,7 @@ public void run() {
 			});
 
 			// API parameters
-			entity.addPart(OpenStreetMapConstants.Api.Gpx.Parameters.FILE, new FileBody(gpxFile, filename, GPX_MIMETYPE, Charset.defaultCharset().name()));
+			entity.addPart(OpenStreetMapConstants.Api.Gpx.Parameters.FILE, new FileBody(gpxFile, filename, DataHelper.MIME_GPX_TYPE, Charset.defaultCharset().name()));
```

Comment:
Since you're using this constant from the Datahelper file, remove the constant "MINE_GPX_TYPE" from this file (see line 45)